src/EventListener/AddRateLimitHeadersListener.php

```diff
@@ -41,7 +31,12 @@ public function onKernelResponse(ResponseEvent $event): void
         $response = $event->getResponse();
         $response->headers->set('x-rate-limit', (string) $storedRateLimit->getLimit());
         $response->headers->set('x-rate-limit-hits', (string) $storedRateLimit->getHits());
-        $response->headers->set('x-rate-limit-until', $storedRateLimit->getValidUntil()->format('c'));
+        $response->headers->set('x-rate-limit-until', $storedRateLimit
+            ->getValidUntil()
+            ->setTimezone(new \DateTimeZone(date_default_timezone_get()))
+            ->format('c')
+        );
+        $response->headers->set('retry-after', (string) ($storedRateLimit->getValidUntil()->getTimestamp() - time()));
```

Comment:
I don't really understand this variable purpose because we already returning the time when on which we can recall the API 🤔 
But why not 🤷

Reply:
@Melfo01 Yes, i have forgot to explain that in the readme. Thanks for the remainder. I have add this header because of it's a rfc from Mozilla ([here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After)) and symfony based his retryable client on this header ([here](https://github.com/symfony/http-client/blob/5.4/RetryableHttpClient.php#L137-L150))